keps/sig-node/4009-add-cdi-devices-to-device-plugin-api/README.md

```diff
@@ -157,10 +157,10 @@ This test case has been added to the existing `e2e_node` tests:
   - DevicePlugin can make a CDI device accessible in a container
 
 Links to test grid:
-- https://testgrid.k8s.io/sig-node-containerd#e2e-cos-device-plugin-gpu
```

Comment:
This change was because we got rid of this job and we are replacing it with the crio job?

Reply:
This is because we now have a job that runs only CDI-related tests, so these links are more appropriate in this KEP in my opinion.

Reply:
My main point about bringing up crio was making sure we had both crio and containerd coverage for GA. Do we have containerd tests for this feature still?

Reply:
No, we don't. I was planning to enable them for containerd 2.0, where CDI is enabled by default. However, I can create a Containerd job similar to the CRI-O one if needed.

Reply:
So what kind of guards do you have in place that this wouldn't break in containerd without these tests? If its covered somewhere I guess its fine.

Reply:
It's partly covered by DRA e2e as it also depends on CDI.